source/includes/steps-enable-authentication-in-shardcluster-nodowntime-auth-config.yaml

```diff
@@ -0,0 +1,31 @@
+stepnum: 1
+level: 4
+title: "Remove ``transitionToAuth`` from the ``mongod`` configuration file."
+ref: modify-config-files
+pre: |
+  
+  Remove the :setting:`security.transitionToAuth` key and its value from the
+  configuration file created during this tutorial.
+  
+  .. code-block:: yaml
+
+     security:
+        keyFile: <path-to-keyfile>
+---
+stepnum: 2
+level: 4
+ref: restart-mongod
+title: "Restart the ``mongod`` with the updated configuration file."
+source:
+  file: steps-enable-authentication-in-shardcluster-nodowntime-transition-config.yaml
+  ref: restart-mongod
+post: |
+
+  At the end of this section the config server replica set should be up and
+  running, fully enforcing authentication.
+  
+  Users and client applications *must* specify authentiation credentials when
```

Comment:
authentication

I think in general we don't expect user/client apps to access the config servers.   I would just remove this sentence since the previous sentence kind of says it all.